e2e-tests/tests/pgvector-extension/01-create-cluster.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+timeout: 10
+commands:
+  - script: |-
+      set -o errexit
+      set -o xtrace
+
+      source ../../functions
+
+      get_cr "pgvector-extension" ${RANDOM} \
+        | yq '.spec.extensions.builtin.pgvector=true' \
```

Comment:
A suggestion: I would deploy cluster with defaults to check that by default it is disabled as expected and then enable/disable to verify it works. WDYT?

Reply:
Yeah, at the same time disabling after I was checking that extension is actually disabled, like 'cleaned' from list of extensions. WDYT?

Reply:
done